ArduPlane/mode.h

```diff
@@ -352,10 +358,14 @@ class ModeAvoidADSB : public Mode
     void update() override;
 
     void navigate() override;
+
+    virtual bool is_guided_mode() const override { return true; }
```

Comment:
this is correct, at least for now, but another step in the onion will be to make this class inherit ModeGuided like it does in copter.